src/switch/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement } from 'lit/decorators.js';
+import { Switch } from './switch';
+import styles from './switch.style';
+
+@customElement('tap-switch')
```

Comment:
Please add JSDoc to your component. Writing JSDoc is necessary for our documentation.

here's a simple example:

```ts
/**
 * @summary A switch component for interacting with boolean values in forms.
 *
 * @prop {boolean} [checked=false] - Indicates whether the switch is checked.
 * @prop {boolean} [disabled=false] - Indicates whether the switch is disabled.
 * @prop {string} [value='on'] - The value of the switch.
 *
 * @csspart [label] - The label of the switch.
 * @csspart [input] - The input element of the switch.
 * @csspart [slider] - The slider of switch.
 *
 */
```

Also provide CSS properties after fixing them